python/lsst/source/injection/bin/make_injection_pipeline.py

```diff
@@ -120,6 +120,32 @@ def build_argparser():
         help="Config override for a task, in the format 'label:key=value'.",
         action="append",
     )
+    parser.add_argument(
+        "-ip",
+        "--input-pipelines",
+        type=str,
+        help="""Location(s) of additional input pipeline definition YAML
+        file(s). Tasks from input_pipelines will be added to the injection
```

Comment:
Tasks from input_pipelines will be... -> Tasks from the additional pipelines will be...